server/modules/data-access/loadBalancerUpstreams.js

```diff
@@ -0,0 +1,93 @@
+/* Copyright (c) Trainline Limited, 2016-2017. All rights reserved. See LICENSE.txt in the project root for license information. */
+
+'use strict';
+
+const LOGICAL_TABLE_NAME = 'InfraConfigLBUpstream';
+
+let {
+  getTableName: physicalTableName
```

Comment:
It's intended to be a more descriptive name for `awsResourceNameProvider.getTableName`.